pages/components/grid/grid.css

```diff
@@ -0,0 +1,28 @@
+.grid-container {
+    margin: 2rem;
+    height: 50vh;
+    display: grid;
+    grid-row: auto auto;
+    grid-column-gap: 1rem;
+    grid-row-gap: 1rem;
+}
+
+.grid-box {
+    border: 2px solid var(--primary-color);
+    background-color: var(--primary-light);
+    padding: 2rem;
+    border-radius: 10px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 40px;
+}
+
+.grid-two {
+    grid-template-columns: 10rem 10rem;
```

Comment:
Can we use fr units here, What if the parent container size will be large, then it will create an issue naa.